scania-digital-design-system/sdds
# AB#854 Feature/footer

**Describe pull-request**  
Add footer style classes

**Solving issue**  
Fixes: [AB#854](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/854)

**How to test**  
_Add description how to test if possible_

- Go to storybook > Patterns > Footer
- To see full page example (Footer + sidemenu + header), go to Patterns > Full Page
Advance:
- Can also check working side menu in sdds demo HTML (use npm pack or npm link to get latest styling)
